<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidSensorToolDigital</class>
 <widget class="QWidget" name="WidSensorToolDigital">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 10px;
}
QFrame#frame:hover{
background-color: rgb(250, 255, 255);
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButton_close">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>31</width>
      <height>70</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Xóa thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_close{
color: rgb(2, 87, 122);
background-color: rgb(25, 149, 173);
border: none;
border-style: outset;
border-width: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
}
QToolButton#toolButton_close:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../chart.qrc">
      <normaloff>:/icon/chart/iconChartClear_On.jpg</normaloff>
      <disabledoff>:/icon/group/iconClear_On.jpg</disabledoff>:/icon/chart/iconChartClear_On.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Unit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>39</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(Unit)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>33</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>13.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>13</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>1</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(3, 87, 123);</string>
    </property>
    <property name="text">
     <string>Cảm biến nhiệt độ</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_info">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Thông tin thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 8px;
border-radius: 8px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../device.qrc">
      <normaloff>:/icon/device/iconDeviceOff_401.jpg</normaloff>:/icon/device/iconDeviceOff_401.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_setup">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>0</y>
      <width>31</width>
      <height>70</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Cài đặt thiết bị</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_setup{
color: rgb(2, 87, 122);
background-color: rgb(25, 149, 173);
border: none;
border-style: outset;
border-width: 0px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
}
QToolButton#toolButton_setup:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../group.qrc">
      <normaloff>:/icon/group/iconSetup_On.jpg</normaloff>
      <disabledoff>:/icon/group/iconSetup_On.jpg</disabledoff>:/icon/group/iconSetup_On.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../group.qrc"/>
  <include location="../device.qrc"/>
  <include location="../chart.qrc"/>
 </resources>
 <connections/>
</ui>
